cosmos/job/drm/DRM_Base.py

```diff
@@ -1,20 +1,64 @@
+from abc import abstractmethod, ABCMeta
+
+
 class DRM(object):
+
     "DRM base class"
+
+    __metaclass__ = ABCMeta
+
     name = None
     poll_interval = 1
+    required_drm_options = None
 
     def __init__(self, jobmanager):
         self.jobmanager = jobmanager
 
+    @classmethod
+    def validate_drm_options(cls, drm_name, drm_options):
+        """Validate that the necessary DRM options have been supplied for the DRM to function.
+        :params str drm_name: The name of the DRM to validate
+        :params dict drm_options: The DRM options to validate
+        """
+        drm_cls = cls.get_drm(drm_name)
```

Comment:
I think no need to use drm_name here,  I think  drm_cls == cls

Reply:
We need to get the subclass from this